src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -58,6 +61,11 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_ADDRESS).isPresent()) {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
+        if (argMultimap.getValue(PREFIX_MODULE).isPresent()) {
+            editPersonDescriptor.setModuleRoleOperation(
+                    parseModuleRolesForEdit(argMultimap.getValue(PREFIX_MODULE).get()).orElse(null));
+
+        }
```

Comment:
Thanks for pointing out! I think since here we only allow 1 r/, following the format of the previous lines of code may be better. It is also clearer and easier to understand.